README.md

```diff
@@ -208,6 +258,49 @@ steps:
 
 ### Remarks
 
+#### Permissions
+
+If you use the
+[`docker-coq`](https://github.com/coq-community/docker-coq) images,
+the container user has UID=GID=1000 while the GitHub action workdir
+has (UID=1001, GID=116).
+This is not an issue when relying on `opam` to build the Coq project.
+Otherwise, you may want to use `sudo` in the container to change the
+permissions. You may also install additional Debian packages.
+
+Typically, this would lead to a workflow specification like this:
+
+```yaml
+runs-on: ubuntu-latest
+strategy:
+  matrix:
+    image:
+      - 'coqorg/coq:dev'
+steps:
+  - uses: actions/checkout@v2
+  - uses: coq-community/docker-coq-action@v1
+    with:
+      opam_file: 'coq-bignums.opam'
+      custom_image: ${{ matrix.image }}
+      before_script: |
+        echo Workaround permission issue
+        sudo chown -R coq:coq .    # <--
+      after_script: |
+        make -C tests
```

Comment:
I think this example is far from ideal, since tests with opam are supposed to be handled by adding something like `run-test: [make "-C" "tests"]` to the opam file, and then making sure running tests is enabled by opam (e.g., by setting `OPAMWITHTEST=true`).

Otherwise, I think everything looks good here.

Reply:
What about a "build with `make`" example, then? Instead of overriding the `after_script` variable, you would override the `script` and `uninstall` variables. As a bonus, this is a real use case (see https://coq.zulipchat.com/#narrow/stream/237977-Coq-users/topic/GitHub.20actions.20CI.20with.20make.20rather.20than.20opam).

Reply:
done, thanks for the suggestion.